pkg/handler/employee/testdata/update_skills/500.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
cái này nên nằm trong file 404.json chứ em. error code not found

Reply:
dạ v chắc em phải fix lại, tại hiện h nó đang là pg trả về user không tồn tại nên nó giống như là internal error á anh